### PR TITLE
Add more resources on RDF, MongoDB and XML

### PR DESCRIPTION
**Pull request by ACSG-64**
This pull request is about:

* [ ] Fixing a bug.
* [ ] Updating documentation.
* [ ] Changing some functionality.
* [x] Other (please explain).

I want to suggest additional resources on RDF and Sparql, MongoDB and XML, all of which are free of charge. I particularly found them very useful while studying the module.

### CODE REVIEW

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md [hunk 1]
```diff
@@ -73,7 +73,14 @@ One two-hour unseen written examination and coursework (Type I)
...
 
 - [Linked Data Engineering (Semantic Web) OpenHPI](https://www.youtube.com/playlist?list=PLoOmvuyo5UAfY6jb46jCpMoqb-dbVewxg). - Prof. Dr. Harald Sack
+- [Working with RDF Graphs in Graph Studio](https://apexapps.oracle.com/pls/apex/r/dbpm/livelabs/view-workshop?wid=3195) - Oracle LiveLabs 
 
 #### SQL
 
 - [The Art of PostgreSQL](https://theartofpostgresql.com/). - _"The book that teaches SQL to developers: Learn to replace thousands of lines of code with simple queries!"_
+
+#### XML
+- [Databases: Semistructured Data](https://www.edx.org/learn/relational-databases/stanford-university-databases-semistructured-data) - Stanford Online
```
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:83]: A blank line should go between the heading and the list.

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md [hunk 2]
```diff
@@ -73,7 +73,14 @@ One two-hour unseen written examination and coursework (Type I)
...
+
+#### Document/JSON
+- [Introduction to MongoDB](https://learn.mongodb.com/learn/learning-path/introduction-to-mongodb) - MongoDB University
```
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:86]: - A blank line should go between the heading and the list.
- The heading `#### Document/JSON` should go before `#### :heart: Notes` to maintain alphabetical order.

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md [hunk 3]
```diff
@@ -73,7 +73,14 @@ One two-hour unseen written examination and coursework (Type I)
 #### Semantic Web
 
 - [Linked Data Engineering (Semantic Web) OpenHPI](https://www.youtube.com/playlist?list=PLoOmvuyo5UAfY6jb46jCpMoqb-dbVewxg). - Prof. Dr. Harald Sack
+- [Working with RDF Graphs in Graph Studio](https://apexapps.oracle.com/pls/apex/r/dbpm/livelabs/view-workshop?wid=3195) - Oracle LiveLabs
```
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:76]:
```suggestion
- [Working with RDF Graphs in Graph Studio](https://apexapps.oracle.com/pls/apex/r/dbpm/livelabs/view-workshop?wid=3195). - Oracle LiveLabs 
```
For consistency with other bullet items.

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md (hunk 1 above)
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:83]:
```suggestion
- [Databases: Semistructured Data](https://www.edx.org/learn/relational-databases/stanford-university-databases-semistructured-data). - Stanford Online
```
For consistency with other bullet items.

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md (hunk 2 above)
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:86]:
```suggestion
- [Introduction to MongoDB](https://learn.mongodb.com/learn/learning-path/introduction-to-mongodb). - MongoDB University
```
For consistency with other bullet items.

**Review by sglavoie — CHANGES_REQUESTED** [review]
Thank you @ACSG-64!

Besides these comments, I would also add the new headings to the table of contents.

A lot of nitpicking... but hopefully that's useful! If not, well someone else will happily merge. 😄

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md (hunk 2 above)
Comment (ACSG-64) [reply modules/level-6/cm-3010-databases-advanced-data-techniques/README.md]: @sglavoie I don't know, don't you think it's better to group the external resources at the end? the `notes` are resources directly related to the course while the other resources are additional external suggestions and so I think it's fine that they are all at the end of the document, just my opinion.

Comment (sglavoie) [reply modules/level-6/cm-3010-databases-advanced-data-techniques/README.md]: Yes, I think you bring a great point. There hasn't really been much of a clear distinction in most of the Markdown files but that's indeed something that should be done. Agreed!

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md [hunk 4]
```diff
@@ -70,10 +72,17 @@ One two-hour unseen written examination and coursework (Type I)
...
 
+#### Semantic Web
 - [Linked Data Engineering (Semantic Web) OpenHPI](https://www.youtube.com/playlist?list=PLoOmvuyo5UAfY6jb46jCpMoqb-dbVewxg). - Prof. Dr. Harald Sack
+- [Working with RDF Graphs in Graph Studio](https://apexapps.oracle.com/pls/apex/r/dbpm/livelabs/view-workshop?wid=3195). - Oracle LiveLabs 
 
 #### SQL
-
 - [The Art of PostgreSQL](https://theartofpostgresql.com/). - _"The book that teaches SQL to developers: Learn to replace thousands of lines of code with simple queries!"_
+
+#### XML
```
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:86]:
```suggestion
#### XML

```

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md [hunk 5]
```diff
@@ -70,10 +72,17 @@ One two-hour unseen written examination and coursework (Type I)
 
 [Reading list for topics 1 to 5](https://github.com/world-class/binary-assets/blob/master/modules/cm3010-dadt/CM3010_reading_list_topics_1-5.pdf).
 
-#### Semantic Web
+<!-- Complimentary external resources -->
+#### Document/JSON
+- [Introduction to MongoDB](https://learn.mongodb.com/learn/learning-path/introduction-to-mongodb). - MongoDB University
 
+#### Semantic Web
```
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:79]:
```suggestion
#### Semantic Web

```

File: modules/level-6/cm-3010-databases-advanced-data-techniques/README.md [hunk 6]
```diff
@@ -70,10 +72,17 @@ One two-hour unseen written examination and coursework (Type I)
 
 [Reading list for topics 1 to 5](https://github.com/world-class/binary-assets/blob/master/modules/cm3010-dadt/CM3010_reading_list_topics_1-5.pdf).
 
-#### Semantic Web
+<!-- Complimentary external resources -->
+#### Document/JSON
```
Comment (sglavoie) [inline modules/level-6/cm-3010-databases-advanced-data-techniques/README.md:76]:
```suggestion
#### Document/JSON

```

**Review by sglavoie — APPROVED** [review]

**Comment by sglavoie** [comment]
@all-contributors please add @ACSG-64 for code

**Pull request merged**